src/app/app.module.ts

```diff
@@ -30,6 +32,8 @@ import { CoursesService } from './courses/courses.service';
     HttpClientModule,
     BrowserAnimationsModule,
     MaterialModule,
+    FormsModule,

```

Comment:
We don't need these here, just in the specific modules that use them (in this case UsersModule which you've already added them to).